### PR TITLE
cli: Display Hostname instead of Peer ID in Volume info

### PR DESCRIPTION
**Pull request by aravindavk**
Also changed the Volume info output Field State to Status

Signed-off-by: Aravinda VK <avishwan@redhat.com>

### CODE REVIEW

File: glustercli/cmd/volume.go [hunk 1]
```diff
@@ -245,15 +245,15 @@ func volumeInfoDisplay(vol api.VolumeGetResp) {
 	fmt.Println("Volume Name:", vol.Name)
 	fmt.Println("Type:", vol.Type)
 	fmt.Println("Volume ID:", vol.ID)
-	fmt.Println("State:", vol.State)
+	fmt.Println("Status:", vol.State)
```
Comment (prashanthpai) [inline glustercli/cmd/volume.go]: I prefer "state" over "status" as it sounds more natural:

`The volume xyz is in started state` as against `The volume xyz is in started status`

But I understand that this change is driven by backwards compatibility.

Comment (aravindavk) [reply glustercli/cmd/volume.go]: Ok, I will revert this back to `State`

Comment (aravindavk) [reply glustercli/cmd/volume.go]: Done

**Comment by aravindavk** [comment]
retest this please

**Pull request merged**